src/Enterspeed.Source.SitecoreCms.V8/Pipelines/Initialize/InitializeEnterspeed.cs

```diff
@@ -156,7 +161,7 @@ private static Item EnsureEnterspeedSiteConfigurationTemplateDataSection(Item en
                 ?? enterspeedSiteConfigTemplateItem.Add("Data", new TemplateID(TemplateIDs.TemplateSection), EnterspeedIDs.Templates.EnterspeedSiteConfigurationDataSectionID);
         }
 
-        private static void EnsureEnterspeedSiteConfigurationDataSectionFields(Item enterspeedSiteConfigSection)
+        private static void EnsureEnterspeedSiteConfigurationDataSectionFields(Item enterspeedSiteConfigSection, Database masterDb)
```

Comment:
Would it makes sense to have some kind of validation/assertion on the parameter? You can pass in web, master or core, but you expect it to be the master database.

Reply:
Not sure. Because it is passed through multiple methods in the implementing class. Then we should look into refactoring the whole class :)

Reply:
ok